src/Providers/MapService.js

```diff
@@ -64,7 +64,8 @@ export var MapServiceProvider = MapService.extend({
         }
         for (var i = 0; i < features.features.length; i++) {
           var feature = features.features[i];
-          layer = (layer) ? layer : response.results[i].layerId;
+          layer = layer || response.results[i].layerId;
```

Comment:
had to change this line manually.  not sure its equivalent.

would you be willing to QA when you have a minute @kneemer?